components/eda/code/eda-bbe/file.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: 'Files, files as events?'
-description: Ballerina enhances file transfer with FTP listeners, reacting to events on servers. SFTP and FTPS options ensure security. Seamlessly integrate file transfer with Ballerina's secure functionality.
+title: 'Files as events'
```

Comment:
The previous title was purposeful. So let's keep it as it is.

Reply:
Done.